v1.34 Changelog

amyblais opened the pull request.

amyblais commented on CHANGELOG.md:
```diff
@@ -1,5 +1,39 @@
+ - **Upgrade to server version v5.19 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.9 has ended and upgrading to server ESR v5.19 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who ...
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
```
Breaking Change?
We've replaced auto-scrolling to the new message line with a "More Messages" button. If the new message line is the  next viewable post, or it is hidden behind the top bar, then in this case we do automatically scroll.

enahum replied:
I don’t consider this breaking

lindalumitchell commented:
FYI: Testing building the apps from this PR since we don't need test builds for it and so this shouldn't be in the way of actual testing. Checking on the new webhook to have Fastlane post in the new QA: Builds channel.

enahum commented on CHANGELOG.md:
```diff
@@ -1,5 +1,40 @@
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue with More Messages button inconsistencies when opening the app from a push notification.
```
Remove, this is from a new feature

amyblais commented:
Found a few other improvements,

enahum commented:
I think this one https://github.com/mattermost/mattermost-mobile/pull/4373 is huge and massive props to @migbot as messages are no longer entering and remaining behind the keyboard.

What about showing the error in the sso screen instead of a long running spinner? https://github.com/mattermost/mattermost-mobile/pull/4609

there is also this one https://github.com/mattermost/mattermost-mobile/pull/4647 filed by you @amyblais

amyblais commented:
https://github.com/mattermost/mattermost-mobile/pull/4373 was already in the changelog.
Added https://github.com/mattermost/mattermost-mobile/pull/4609 (originally thought it was a temporary fix).
From the PR description I thought https://github.com/mattermost/mattermost-mobile/pull/4647 was a regression, but I added it.

amyblais commented:
@enahum This can be merged
